include/multipass/platform.h

```diff
@@ -56,6 +56,7 @@ VirtualMachineFactory::UPtr vm_backend(const Path& data_dir);
 logging::Logger::UPtr make_logger(logging::Level level);
 UpdatePrompt::UPtr make_update_prompt();
 std::unique_ptr<Process> make_sshfs_server_process(const SSHFSServerConfig& config);
+std::unique_ptr<Process> make_qemuimg_process(const QStringList& parameters);
```

Comment:
Couldn't this be generic instead?
```suggestion
std::unique_ptr<Process> make_process(const ProcessSpec& spec, const QStringList& parameters);
```

(or an appropriate smart pointer version of the above)

Reply:
Good point, done.